sdk/storage/azblob/README.md

```diff
@@ -1,4 +1,6 @@
 # Azure Blob Storage module for Go
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/go/go%20-%20azdatalake%20-%20ci?branchName=main)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=2854&branchName=main)
```

Comment:
I don't see GO reference part in the readme as mentioned in the PR description.

Reply:
The commit for go-reference was not checked in. Checked in now.